rocky/assets/js/renderNormalizerOutputOOIs.js

```diff
@@ -0,0 +1,45 @@
+
+const buttons = document.querySelectorAll(".expando-button.normalizer-list-table-row");
+
+buttons.forEach((button) => {
+  const raw_task_id = button.closest('tr').getAttribute('data-task-id');
+  const task_id = button.closest('tr').getAttribute('data-task-id').replace(/-/g, "");
+  const json_url = window.location.href + "/" + task_id;
```

Comment:
If this JS is going to be used on other pages too, this maybe just needs to be a constant?

Reply:
Not sure what you mean. The task_id is only known within the context of the button, right?